metadata.rb

```diff
@@ -31,8 +31,8 @@
 depends 'chef_handler'
 
 # For apt and yum repositories
-depends 'apt', '~> 2.8'
-depends 'yum', '~> 3.8'
+depends 'apt', '~> 2.7'
```

Comment:
My guess is this is the cookbook version they were using, and probably the old version was just chosen b/c it was the version SimpleFinance was using.  I had a look at the Changelogs for `apt` and `yum` and it seems okay to open these up (its a pessimistic match so it will match anything >=2.7 and  < 3.0).